src/Action/action_lib/action_u1_dm.jl

```diff
@@ -56,8 +56,18 @@ function init_action!(
     return nothing
 end
 
+# set repulsion for su2 symmetry
+function set_repulsion!(
+    A :: Float64,
+    a :: Action_u1_dm
+    ) ::Nothing
 
+    # init bare action on-site with level repulsion for Γxx
+    a.Γ[1].bare[1] = A / 4.0 / (2.0 * a.S)
```

Comment:
Does this work? The u1-dm type has no field `S` as this construction only works for Heisenberg models.

Reply:
```suggestion
    a.Γ[1].bare[1] = A / 4.0
```
Somehow I missed this actually :/ thought I tested this...